### PR TITLE
initial german translations

### PR DESCRIPTION
**Pull request by rschnapka**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -54,3 +54,4 @@ docs/_build/
 # Backup files
 *~
 *.swp
+/.settings/
```
Comment (pedrobaeza) [inline .gitignore:57]: Why do you change this?

**Comment by pedrobaeza** [comment]
:+1:

File: .gitignore (hunk 1 above)
Comment (rschnapka) [inline .gitignore:57]: I have no clue - wasn't intentionally...

**Comment by yvaucher** [comment]
Please remove change on .gitignore

**Comment by yvaucher** [comment]
Merged at https://github.com/OCA/sale-financial/commit/0e85b1ef881aad827f8ff85b27c9b9d6531b1824 removing change in `.gitignore`

**Pull request closed without merging**